container/functional/keylime_verifier_registrar_container-basic-attestation/test.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+#How to run it
+#tmt -c distro=rhel-9.1 -c agent=rust run plan --default discover -h fmf -t /setup/configure_kernel_ima_module/ima_policy_simple -t /functional/keylime_agent_container-basic-attestation -vv provision --how=connect --guest=testvm --user root prepare execute --how tmt --interactive login finish
+#Machine should have /dev/tpm0 or /dev/tpmrm0 device
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        # update /etc/keylime.conf
+        limeBackupConfig
+        #getting ip of host
+        #IP_AGENT=$( hostname -I | awk '{ print $1 }' )
+        CONT_NETWORK_NAME="container_network"
+        IP_VERIFIER="172.18.0.4"
+        IP_REGISTRAR="172.18.0.8"
+        IP_AGENT="172.18.0.12"
+        #create network for containers
+        rlRun "limeconCreateNetwork ${CONT_NETWORK_NAME} 172.18.0.0/16"
+
+        #prepare verifier container
+        rlRun "limeUpdateConf verifier ip $IP_VERIFIER"
+        rlRun "limeUpdateConf verifier registrar_ip $IP_REGISTRAR"
+        #for log purposes, when agent fail, we need see verifier log, that attestation failed
+        rlRun "limeUpdateConf verifier log_destination stream"
+
+        # prepare registrar container
+        rlRun "limeUpdateConf registrar ip $IP_REGISTRAR"
+
+        #build verifier container
+        TAG_VERIFIER="verifier_image"  
+        podman build -t=$TAG_VERIFIER --file "${limeLibraryDir}/Dockerfile.verifier" .   
+        
+        #build registrar container
+        TAG_REGISTRAR="registrar_image"  
+        podman build -t=$TAG_REGISTRAR --file "${limeLibraryDir}/Dockerfile.registrar" .
+
+        #run verifier container
+        CONT_VERIFIER="verifier_container"
+        rlRun "podman run -d --name $CONT_VERIFIER --net $CONT_NETWORK_NAME --ip $IP_VERIFIER -p 8881:8881 --volume=/etc/keylime/:/etc/keylime/ localhost/$TAG_VERIFIER /usr/bin/keylime_verifier"
+        rlRun "limeWaitForVerifier"
```

Comment:
This works only because we have published its port previously. Maybe we should introduce IP as an optional argument so that we would be waiting for an opened port on the given IP (and not always/only localhost).